composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "bcgov-theme/bcgov-wordpress-block-theme",
     "description": "Block Theme for BC Gov",
-    "version": "1.2.19",
+    "version": "1.2.20",
```

Comment:
technically this is a feature not a patch https://semver.org/

Reply:
fair enough